src/Symfony/Bridge/Twig/Tests/Command/DebugCommandTest.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Tests\Command;
+
+use PHPUnit\Framework\TestCase;
+use Symfony\Bridge\Twig\Command\DebugCommand;
+use Symfony\Component\Console\Application;
+use Symfony\Component\Console\Tester\CommandTester;
+use Twig\Loader\FilesystemLoader;
+use Twig\Environment;
+
+class DebugCommandTest extends TestCase
+{
+    public function testDebugCommand()
+    {
+        $tester = $this->createCommandTester();
+        $ret = $tester->execute(array(), array('decorated' => false));
+
+        $this->assertEquals(0, $ret, 'Returns 0 in case of success');
+        $this->assertContains('Functions', trim($tester->getDisplay()));
+    }
+
+    public function testLoaderPaths()
+    {
+        $tester = $this->createCommandTester(array(
+            'Acme' => array('extractor', 'extractor'),
+            '!Acme' => array('extractor', 'extractor'),
```

Comment:
I suggest using different paths for some of them, to make the test a bit more realistic

Reply:
Right, but that would require create an empty directory (with some useless templates) into `Fixtures` (root path), anyway they are fakes paths to show the line separator, it still worth it?

Reply:
I added a comment to avoid any confusion when reading the code.